src/lib/ens-helpers.ts

```diff
@@ -1,17 +1,16 @@
+import { bytesToPacket } from "@ensdomains/ensjs/utils";
 import { type Hex, concat, keccak256, namehash, toHex } from "viem";
 
-// TODO: pull from ens utils lib or something
 export const NAMEHASH_ZERO = namehash("");
 export const NAMEHASH_ETH = namehash("eth");
```

Comment:
This returns the id of the node for the "eth" tld. Perhaps consider giving this a name such as `ETH_TLD_NODE`.